src/apps/settings/SettingsVRCTab.svelte

```diff
@@ -0,0 +1,60 @@
+<script>
+    import { getContext } from "svelte";
```

Comment:
I think we should generalize this to an extra content tab instead of just being specific to VRC.